src/plugins/sockets/inGameEvents.js

```diff
@@ -196,6 +196,7 @@ export async function handleInGameEvents(evData) {
         !evData.victory.gameOver && evData.playedBy !== gameStore.myPNum;
       gameStore.waitingForOpponentToStalemate = !evData.victory.gameOver && evData.playedBy === gameStore.myPNum;
       break;
+    case SocketEvent.ACCEPT_STALEMATE:
```

Comment:
Added this case to allow clearing both dialog and overlay when stalemate is accepted. They used to both stay open in the legacy API because it looked like a request for a stalemate initiation, rather than to accept one and end the game.